### PR TITLE
feat(zimbra): add mailing list tab

### PR DESCRIPTION
**Pull request by stif59100**
ref:MANAGER-14868

| Question         | Answer
| ---------------- | ---
| Branch?          | feat/zimbra-ga
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-14868
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description

I added tab mailing list for zimbra

### CODE REVIEW

**Review by tristanwagner — APPROVED** [review]

File: packages/manager/apps/zimbra/src/components/layout-helpers/Dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ import {
 
 // import { OdsHTMLAnchorElementTarget } from '@ovhcloud/ods-common-core';
 import { useTranslation } from 'react-i18next';
-import { ShellContext } from '@ovh-ux/manager-react-shell-client';
+// import { ShellContext } from '@ovh-ux/manager-react-shell-client';
```
Comment (ghyenne) [inline packages/manager/apps/zimbra/src/components/layout-helpers/Dashboard/Dashboard.tsx:17]: Why need comment this ?

File: packages/manager/apps/zimbra/src/components/layout-helpers/Dashboard/Dashboard.tsx [hunk 2]
```diff
@@ -76,6 +76,12 @@ export const Dashboard: React.FC = () => {
       to: `${basePath}/email_accounts`,
       pathMatchers: computePathMatchers([urls.email_accounts]),
     },
+    {
+      name: 'mailing_list',
+      title: t('zimbra_dashboard_mailing_list'),
+      to: `${basePath}/mailing_list`,
```
Comment (ghyenne) [inline packages/manager/apps/zimbra/src/components/layout-helpers/Dashboard/Dashboard.tsx:82]:
```suggestion
      to: `${basePath}/mailing_lists`,
```
The rest of url of tabs it's in plurial

File: packages/manager/apps/zimbra/src/routes/routes.tsx [hunk 3]
```diff
@@ -141,6 +141,12 @@ export const Routes: any = [
               },
             ],
           },
+          {
+            path: 'mailing_list',
```
Comment (ghyenne) [inline packages/manager/apps/zimbra/src/routes/routes.tsx:145]: Same https://github.com/ovh/manager/pull/12992#discussion_r1751495251

File: packages/manager/apps/zimbra/src/routes/routes.constants.ts [hunk 4]
```diff
@@ -12,4 +12,5 @@ export const urls = {
   email_accounts_alias: '/:serviceName/email_accounts/alias',
   email_accounts_alias_add: '/:serviceName/email_accounts/alias/add',
   email_accounts_alias_delete: '/:serviceName/email_accounts/alias/delete',
+  mailing_list: '/:serviceName/mailing_list',
```
Comment (ghyenne) [inline packages/manager/apps/zimbra/src/routes/routes.constants.ts:15]: Same https://github.com/ovh/manager/pull/12992#discussion_r1751495251

**Review by ghyenne — APPROVED** [review]

**Pull request merged**